preprocessors/openstreetmap/osm_service.py

```diff
@@ -27,6 +27,11 @@ def create_bbox_coordinates(distance, lat, lon):
     """ Convert lat/lon from radians back to degrees """
     lat_min, lon_min = degrees(lat_min), degrees(lon_min)
     lat_max, lon_max = degrees(lat_max), degrees(lon_max)
+    """ Convert lat/lon to 7 decimal places """
```

Comment:
Could you explain why it is necessary to round here?

Reply:
The approximation to seven decimal places does not in any way introduce an additional effect. It's just for uniformity’s sake since every other float variable (i.e., lat/lon) is represented in 7d.p.  So, I feel it looks better than having long float numbers as in ```"bounds": {
			"latitude": {
				"min": 49.89653157839408,
				"max": 49.898330221605924
			},
			"longitude": {
				"min": -97.20479051975164,
				"max": -97.20199828024836 ```

Reply:
Look doesn't terribly bother me since this isn't meant for people to directly work with outside of prototyping and debugging. It also probably isn't enough to lead to any significant changes, so it's fine staying. If there were a reason for it other than aesthetics, it would just need to be documented.